### DISCUSSION TITLE
Playlists: Custom images

### DISCUSSION THREAD

**Discussion by Floydimus02** [tidal-music/discussions#109] [Ideas & Feature requests]
Hello!

Playlists considers album art of top four songs as playlist. This confuses me, as a listener, to a) locate and b) manage playlists and I end up spending more time in solving the aforementioned problems than listening to music.

While this may be a cosmetic change with no major metric impact, can we please consider prioritising this alongside other cosmetic issues?

Shoutout to the team at Tidal for fantastic app; I just LOVE Tidal.

**Comment by ungive** [comment]
I've stumbled upon this too before and would love to see this as an option in a future update!

**Comment by fmalhe** [comment]
Hi there!

Thanks so much for the feedback and the shoutout! We’re actively looking at ways to improve playlist visuals to make them easier to manage. Quick question: would you prefer an option to upload custom cover art, or would a simpler default cover work for you?

Your input is super helpful—thanks for sharing!

**Reply by Floydimus02** [reply]
> Thanks so much for the feedback and the shoutout!

You're most welcome :D

> Quick question: would you prefer an option to upload custom cover art, or would a simpler default cover work for you?

This triggered my product manager brain. Haha!

What do you mean by simpler/default cover? Does it mean I can pick from the artists cover images/album arts from the songs in the playlists? Or does it mean that Tidal generates a gradient image and puts as a cover art.

In general, as a user, I would prefer simpler approach where I can just pick from a suggested set of images (uploading is cumbersome, though provides higher control).

**Reply by duttonst** [reply]
Just to add my 2c, being able to upload playlist artwork (like Spotify allows) would be my preferred option. It's a big area where third-party apps can add value.

**Reply by Floydimus02** [reply]
@duttonst I wonder whether Tidal (relatively a smaller service as compared to Spotify) wants to open up for third parties.

**Reply by duttonst** [reply]
> @duttonst I wonder whether Tidal (relatively a smaller service as compared to Spotify) wants to open up for third parties.

I assumed that was why they had spent all this effort building a public API with a (planned) app approval process?

**Reply by fmalhe** [reply]
Thanks for all the great feedback and ideas! Customization is something we know is important, and we're always keeping user input in mind as we look at potential improvements. While I can’t promise specifics yet, your suggestions—like choosing from a set of images—are really valuable as we explore possible updates.

We’ll definitely keep you posted on any developments. Thanks again for sharing your thoughts!

**Reply by ungive** [reply]
Last time I wanted to set a custom playlist cover I wanted to upload an image. It was actually a playlist I moved over from another streaming service which allows uploads of custom covers images. I would have an easier time finding that playlist as I go back to it every once in a while and I associate that picture with it.

**Reply by Floydimus02** [reply]
> I assumed that was why they had spent all this effort building a public API with a (planned) app approval process?

@duttonst my bad; I was unaware of public APIs.

> are really valuable as we explore possible updates.
> We’ll definitely keep you posted on any developments.

@fmalhe you are most welcome and thank you; just read the news and it was disheartening to know that reorg is underway. I hope things are fine with you.

**Reply by RebellionBritney** [reply]
> Hi there!
> 
> Thanks so much for the feedback and the shoutout! We’re actively looking at ways to improve playlist visuals to make them easier to manage. Quick question: would you prefer an option to upload custom cover art, or would a simpler default cover work for you?
> 
> Your input is super helpful—thanks for sharing!

Upload custom cover art 🙏

**Reply by Dominic-t3ch** [reply]
100% agree with this request. I was a Sporify user for few years. Then switched to Qobuz and since 4 months switched to Tidal. And this one became my favorite. Great sound, nice looking with a useful home page and discovery algorithm. But the only thing that miss me is the possibility to upload custom cover art image for playlist.

I created many compilation playlists for my favorite artists and I would like to upload a custom image for these playlist.

**Reply by RebellionBritney** [reply]
Please, @fmalhe we need a GitHub for Tidal app, we want to give this kind of feedback and suggestions... doing it via TIDAL Support is useless, especially since we don't get an answer and it's much better if we directly give you and help you with this kind of stuff. Is a win-win for everyone, there's a lot we can help each other to make TIDAL a great service

**Reply by almostarden** [reply]
hi there! just adding my two cents of agreement: recently migrated from Spotify and, so far, the only thing that feels inferior is the lack of custom uploads/custom playlist art. it's not just about customisation (for me at least), it makes finding the playlists easier (i'm forgetful, names are a concept), makes them feel complete & aesthetically pleasing to share, and helps me differentiate playlists from each other, especially if there's an overlap of the same songs. <3 if you guys did this, you'd have quite a few ride or dies for life!

**Reply by sonalder-darlene** [reply]
I am also disapointed about the lack of customization of my playlist, having a name is great, a description is cool but a cover is super important as people juge books by their covers, movies by their poster and so on. It refelects the vibe of the playlist.
Official Tidal's playlists do have custom covers so why your users couldn't ?
Honestly this is the biggest drawback of your service in my opinion after six months of use

**Comment by fmalhe** [comment]
Hi everyone,

Thanks for all the input shared here — I realize this thread has been quiet for a while, and I appreciate your patience.

Just to clarify: this GitHub Discussions space is meant for technical topics related to the [TIDAL Developer Platform](https://developer.tidal.com/), such as our APIs and SDKs. For feature requests related to the TIDAL app itself — like uploading custom playlist covers — the best way to share feedback is through [TIDAL Support](https://support.tidal.com/), so it can reach the right team internally.

Apologies if my earlier message gave the impression that this was the right place to continue that kind of discussion.

**Comment by Floydimus02** [comment]
Oohho!! This was implemented. Kudos to @fmalhe and the team at Tidal; thank you!

**Comment by john-kick** [comment]
Is there any way to upload a custom cover art for playlists using the API? Looking at the [API reference](https://tidal-music.github.io/tidal-api-reference), it's possible to get the current cover art of playlists using `GET /playlists/{id}/relationships/coverArt`, but there is no corresponding `POST` endpoint. Did I miss something? Or is this something planned in the future?

**Comment by CharliePalm** [comment]
+1 to the above - are there any plans to include the ability to modify playlist cover photos by the exposed developer API?
